ui/js/graphs.js

```diff
@@ -468,16 +478,22 @@ perf.controller('GraphsCtrl', [
                              label: series.projectName + " " + series.name,
                              data: [],
                              resultSetData: [],
+                             customeasure:$scope.myMeasure,
```

Comment:
Just saw the PR in passing :) Wondering if perhaps we'd want to use `customMeasure` for consistency.